sklearn/metrics/cluster/__init__.py

```diff
@@ -33,14 +33,14 @@
 )
 
 __all__ = [
+    "_entropy",
```

Comment:
I would also remove it from this ``__all__`` since private functions should not populate public namespaces.

Reply:
Done, let's see what others think.